jupyter-on-gke/README.md

```diff
@@ -0,0 +1,119 @@
+# Ray on GKE
```

Comment:
Jupyter on GKE?

Reply:
I think you can keep this file empty, I can add a readme for Jupyterhub once this PR is merged.